components/Form/index.js

```diff
@@ -69,8 +70,14 @@ export default function Form({
         />
         {isEditMode && (
           <>
-            <label htmlFor="notes">Notes, use case ...</label>
-            <textarea></textarea>
+            <label htmlFor="notes">Add notes ...</label>
+            <textarea
+              type="text"
```

Comment:
the attribute `type` does nothing for a textarea, you can remove this line of code

```suggestion
```

Reply:
Done